### PR TITLE
Use latest OPC stack

### PR DESCRIPTION
**Pull request by mregen**
- use latest 363 bits, now supporting netstandard2.1
- include preview feed of OPC Foundation for development
- fix also a few build errors due to warnings

### CODE REVIEW

File: components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Transport/Http/WebSocketChannelListener.cs [hunk 1]
```diff
@@ -213,6 +214,18 @@ public class WebSocketChannelListener : ITcpChannelListener,
                 }));
         }
 
+        bool ITcpChannelListener.ReconnectToExistingChannel(IMessageSocket socket, uint requestId, uint sequenceNumber, uint channelId, X509Certificate2 clientCertificate, ChannelToken token, OpenSecureChannelRequest request) {
```
Comment (mregen) [inline components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Transport/Http/WebSocketChannelListener.cs:217]: add inheritdoc

File: NuGet.Config [hunk 2]
```diff
@@ -3,5 +3,6 @@
   <packageSources>
     <add key="nugetv3" value="https://api.nuget.org/v3/index.json" />
     <add key="myget" value="https://www.myget.org/F/azureiiot/api/v3/index.json" />
+    <add key="opcua-preview" value="https://opcfoundation.pkgs.visualstudio.com/opcua-netstandard/_packaging/opcua-preview/nuget/v3/index.json" />
```
Comment (barnstee) [inline NuGet.Config:6]: How do we make sure we don't accidentally pull in preview packages? Are they marked "-preview"?

**Review by barnstee — APPROVED** [review]

File: NuGet.Config (hunk 2 above)
Comment (mregen) [reply NuGet.Config:6]: yep, they are all marked -preview

**Review by koepalex — APPROVED** [review]
:shipit:

**Pull request merged**